slidez-fe-core/src/common/components/forms/LoginForm.tsx

```diff
@@ -106,13 +108,14 @@ const LoginForm = ({ onLogin, onLoginWithGoogle }: LoginProps) => {
                     />
                 </div>
             </div>
-            <div className='form-row buttons-row'>
-                <button
+            <div className='form-row' />
+            <div className='form-row'>
```

Comment:
Should there be two identical divs?

Reply:
It was fixed in develop with styles, so please merge correctly this part